MapleServer2/PacketHandlers/Game/UserSyncHandler.cs

```diff
@@ -39,9 +50,33 @@ public override void Handle(GameSession session, PacketReader packet)
 
             Packet syncPacket = SyncStatePacket.UserSync(session.FieldPlayer, syncStates);
             session.FieldManager.BroadcastPacket(syncPacket, session);
+
+            if ((session.Player.SafeCoord - syncStates[0].Coord.ToFloat()).Length() > 200 && session.FieldPlayer.Coord.Z == syncStates[0].Coord.ToFloat().Z && session.Player.SafeCoord != session.FieldPlayer.Coord.ClosestBlock() && !session.Player.OnAirMount) // Save last coord if player is not falling
```

Comment:
Maybe change `(session.Player.SafeCoord - syncStates[0].Coord.ToFloat()).Length() > 200` to a bool variable

Reply:
`&& session.Player.SafeCoord != session.FieldPlayer.Coord.ClosestBlock()` seems unnecessary